common/eth2_network_config/built_in_network_configs/mainnet/config.yaml

```
@@ -89,4 +89,15 @@ DEPOSIT_CONTRACT_ADDRESS: 0x00000000219ab540356cBB839Cbe05303d7705Fa
 
 # Network
 # ---------------------------------------------------------------
-SUBNETS_PER_NODE: 2
\ No newline at end of file
+SUBNETS_PER_NODE: 2
+GOSSIP_MAX_SIZE: 10485760
+MAX_REQUEST_BLOCKS: 1024
+MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
+MAX_CHUNK_SIZE: 1048576
```

```suggestion
MAX_CHUNK_SIZE: 10485760
```
From https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L116